app/src/main/java/de/westnordost/streetcomplete/quests/parking_access/AddParkingAccess.java

```diff
@@ -0,0 +1,41 @@
+package de.westnordost.streetcomplete.quests.parking_access;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddParkingAccess extends SimpleOverpassQuestType
+{
+	@Inject public AddParkingAccess(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override
+	protected String getTagFilters() {
+		return "nodes, ways with amenity=parking and (!access or access=unknown)";
```

Comment:
BTW why do you use different queries here and [here](https://github.com/westnordost/StreetComplete/pull/825/files#diff-21e98c586106a4fcb30e234c8c81c5a9R25)? Also, the parking type has [yet another query](https://github.com/westnordost/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/parking_type/AddParkingType.java#L24)…
Would not it be better to unify this? Maybe even in a base class?